src/main/java/Alexer.java

```diff
@@ -0,0 +1,224 @@
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+import java.util.stream.Collectors;
+
+public class Alexer {
+    private static final String GOODBYE = "Goodbye! If you ever want to chat again, I'll be here.\nHave a great day!";
+    private static final String BREAK = "____________________________________________________________";
+
+    public static final String NAME = "Alexer";
+
+    private final Scanner scanner = new Scanner(System.in);
+
+    private final TaskManager tasks = new TaskManager();
```

Comment:
Maybe rename this to taskManager? I would think tasks would represent the tasks themselves e.g. an arraylist.

Reply:
Thanks for the suggestion, will take that into consideration!